### ISSUE TITLE
How to update saved model

### ISSUE THREAD

**Issue by PonraJS-21** [tensorflow/tensorflow#37280]
I would like to update my pretrained mode, which means the data of model should exist and new data to add.

**Comment by ravikyram** [comment]
@PonraJS-21 
Can you please go through the [link ](https://www.tensorflow.org/tutorials/images/transfer_learning)and see if it helps you.

This question is better asked on StackOverflow since it is not a bug or feature request. There is also a larger community that reads questions there and provide better and faster support for such issues. Thanks!

**Comment by Stealers** [comment]
y_pred = inference(input_1=tf.constant(image_batch,dtype=tf.float32), input_4=tf.constant(anchors_batch,dtype=tf.float32))#, tf.constant(anchors_batch,dtype=tf.float32)

I made it!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
